examples/experimental/dagster-airlift/dagster_airlift/core/task_asset.py

```diff
@@ -1,21 +1,58 @@
-from typing import Any, Mapping, Optional
+from typing import Any, List, Mapping, NamedTuple, Optional
 
-from dagster import JsonMetadataValue
+from dagster import (
+    JsonMetadataValue,
+    _check as check,
+)
+from dagster._core.definitions.asset_spec import AssetSpec
 from dagster._core.definitions.metadata.metadata_value import UrlMetadataValue
+from dagster._record import record
 
 from dagster_airlift.constants import MIGRATED_TAG
 from dagster_airlift.core.airflow_instance import TaskInfo
 from dagster_airlift.core.serialization.serialized_data import SerializedAssetKeyScopedAirflowData
 from dagster_airlift.core.utils import airflow_kind_dict
 
 
+class TaskHandle(NamedTuple):
+    dag_id: str
+    task_id: str
+
+
+@record
+class FetchedAirflowTask:
```

Comment:
FetchedAirflowTaskData? I just like to make it clear that it doesn't have any operational component beyond metadata

Reply:
Sgtm. I will rename at tip of the stack